ADO.NET/OAuthSampleApp/README.md

```diff
@@ -2,28 +2,37 @@
 
 The Vertica ADO.NET driver now allows connecting through OAuth access tokens. 
 
-This sample application was built in .NET 6.0 and can be run in .NET Framework 4.6+ or .NET Core 3.1+.
+This application simulates what you would want your application to do when connecting with access and refresh tokens. We store the tokens as environment variables but they can be stored in any fashion. The logic of the program is:
+1. If the access token is set, attempt to connect.
+2. If the access token is not set and there is a refresh token, attempt to get a new access token.
+3. If neither is set (i.e. running the app for the first time), set the access and refresh tokens through the direct access (password) grant.
+4. Attempt to connect with the access token in the connection string.
+5. If the access token is invalid/expired, use the refresh token to get a new one and try again.
+6. If the refresh token is expired, use the direct access flow to get new tokens and try again.
 
 # Prerequisites
```

Comment:
Please look at the prerequisites you added for the browser flow sample app and use the relevant ones here

- An IDP (tested with Keycloak)
- Do they need the .NET core?
- ADO.NET driver (at least 24.3)?
- Vertica (at least 12.1)

Reply:
This information is right below